tests/rpc.rs

```diff
@@ -436,3 +444,53 @@ async fn test_getClassHashAt() -> Result<(), Error> {
     );
     Ok(())
 }
+
+#[tokio::test]
+async fn test_declare_deploy_account_already_existing() -> Result<(), Error> {
+    let ctx = setup!("sepolia");
```

Comment:
if both DECLARE and DEPLOY_ACCOUNT transactions are expected to fail, what is the point of running it on sepolia only?

Reply:
Because this account already exists on sepolia, it does not on mainnet. Calling `addDeclareTransaction` and `addDeployAccountTransaction` would start the process of adding new account to the mainnet which I think is not suitable for a test case at this point. Maybe some time when we will have budget of starks for mainnet, it could be taken into account.

Reply:
Forgot to write in description that also `getClass` is called that verifies that account exists on sepolia. This way if somehow setup comes to mainnet instead of sepolia in this specific test case, this should fail first before any bamboozle declare/deploy calls would happen.

Reply:
> adding new account to the mainnet

> is not suitable for a test case at this point

Not trying to be funny here, just thinking what this test actually tests.

The way I see it, it tests that it is impossible to declare & deploy an existing account. If the test fails, it means that somehow the existing account go declared & deployed again. So it actually tests Starknet.

What we neet to test is that the it is possible (using owr code) to deploy a new account. We might want to run this test on demand (idk, once a month), on provided input (secret key). Then if such test fails, it means that probably something is wrong with our code, and we can verify that (e.g. run A/B tests with/without Beerus as a proxy). Such test would make sense to run on testnet (to fee in SepoliaETH not it real ETH).

Reply:
> Since this transaction needs gas and prefunded wallets, it has been decided to have this test on sepolia with an already deployed same account.

Prefunded wallet on Sepolia is not a problem, rather a pre-requisite for the test. So decision to run the test against deployed account renders this test pointless, unfortunately.